corpora/csip/metadata/mets-root/CSIP4/testCase.xml

```diff
@@ -1,23 +1,161 @@
+          <path></path>
+          <description>
+            METS.xml file has mets/@csip:CONTENTINFORMATIONTYPE = "random_string_du92Neib57zjMWren".
+          </description>
+        </package>
+        <!--  It seems most efficient to have one valid package for all SHOULD 
+              requirements - there are about 20 of them, so the alternative would
+              be to have 20 rule-specific valid packages. -->
+        <!--  Ideally, there should be a valid IP for each specific vocabulary 
+              value, as this is the only certain way to test that the validator 
+              accepts any one of them. This should be automated as 
+              VocabularyContentInformationTypeSpecification currently allows 7
+              different values. -->
+        <package name="minimal_IP_with_1_rep_plus_mets-xml_SHOULD_nodes" isValid="TRUE">
+          <path></path>
+          <description>
+            Minimal IP that matches all MUST requirements, plus all SHOULD requirements (both for elements and attributes).
+          </description>
+        </package>
+      </corpusPackages>
+    </rule>
+    <rule id="4">
+      <!--  Rule 5 (the check for existence and value) covers it, but to make the
+            error messages more precise, it makes sense to include a separate
+            rule for existence only. -->
+      <description>If mets/@csip:CONTENTINFORMATIONTYPE = "OTHER" then mets/@csip:OTHERCONTENTINFORMATIONTYPE MUST exist.</description>
+      <error level="ERROR">
+        <message>
+          mets/@csip:OTHERCONTENTINFORMATIONTYPE attribute does not exist. MUST be: If mets/@csip:CONTENTINFORMATIONTYPE = "OTHER" then mets/@csip:OTHERCONTENTINFORMATIONTYPE MUST exist and have a value.
+        </message>
+      </error>
+      <corpusPackages>
+        <package name="mets-xml_mets_csip-CONTENTINFORMATIONTYPE_OTHER_and_csip-OTHERCONTENTINFORMATIONTYPE_not_exist" isValid="FALSE">
+          <path></path>
+          <description>
+            METS.xml file has mets/@csip:CONTENTINFORMATIONTYPE = "OTHER", mets/@csip:OTHERCONTENTINFORMATIONTYPE attribute does not exist.
+          </description>
+        </package>
+        <package name="minimal_IP_with_1_rep_mets_csip-CONTENTINFORMATIONTYPE_OTHER" isValid="TRUE">
+          <path></path>
+          <description>
+            Minimal IP that matches all MUST requirements, mets/@csip:CONTENTINFORMATIONTYPE = "OTHER", mets/@csip:OTHERCONTENTINFORMATIONTYPE = "SIARDUK".
+          </description>
+        </package>
+      </corpusPackages>
+    </rule>
+    <rule id="5">
+      <description>If mets/@csip:CONTENTINFORMATIONTYPE = "OTHER" then mets/@csip:OTHERCONTENTINFORMATIONTYPE MUST exist and have a value.</description>
+      <error level="ERROR">
+        <message>
+          mets/@csip:OTHERCONTENTINFORMATIONTYPE attribute has no value. MUST be: If mets/@csip:CONTENTINFORMATIONTYPE = "OTHER" then mets/@csip:OTHERCONTENTINFORMATIONTYPE MUST exist and have a value.
```

Comment:
What about this message:
mets/@csip:OTHERCONTENTINFORMATIONTYPE attribute has no value. If mets/@csip:CONTENTINFORMATIONTYPE = "OTHER" then mets/@csip:OTHERCONTENTINFORMATIONTYPE MUST exist and have a value.